great-idea/services.html

```diff
@@ -1,87 +1,110 @@
 <!-- copy and paste your index.html page structure in here, markup the content provided after that -->
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+    <body>
+    
+
+        <!-- Copy and paste your HTML from the first UI project here -->
+        <div class="container">
+            <header>
+                <nav class="navigation">
+                    <a href="services.html">Services</a>
+                    <a href="">Product</a>
+                    <a href="">Vision</a>
+                    <a href="">Features</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                </nav>
+                 <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> </a>
+            </header>
+    
+            <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+            
+            <section class="services-top">
+                <h2>Services</h2>
+                <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p> 
+            </section> 
+
+            <hr>
+            <div class="boxes">
+            <section class="box">
+                <h2>Digital Design</h2>
+                <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                eleifend. Mauris euismod facilisis iaculis.</p> 
+                <button class="learn-more">Learn More</button>
```

Comment:
When styling buttons you can generally just call them by button. Most of the time they don't need a class assigned.